module/signature/threshold.go

```diff
@@ -95,7 +95,13 @@ func EnoughThresholdShares(size int, shares int) (bool, error) {
 // signature for the group of the given size. The indices represent the index for ech signature share
 // within the DKG algorithm.
 func (tp *ThresholdProvider) Reconstruct(size uint, shares []crypto.Signature, indices []uint) (crypto.Signature, error) {
+	return CombineThresholdShares(size, shares, indices)
```

Comment:
I guess we should delete `CombineThresholdShares` and just keep `Reconstruct` (the name we would like to keep)

Reply:
Is it OK if I leave this for the crypto refactor?

Reply:
What I meant is to not use the extra wrapper. We can just remove [these lines](https://github.com/onflow/flow-go/blob/feature/epochs/module/signature/threshold.go#L98-L104).

Reply:
Updated in https://github.com/onflow/flow-go/pull/295/commits/a75e9339e2c8c8959d27344a01985a604810179d